packages/p2p/src/components/nickname/nickname-form.jsx

```diff
@@ -67,7 +67,7 @@ const NicknameForm = ({ handleClose }) => {
                 </div>
             )}
             <Formik validate={validatePopup} initialValues={{ nickname: '' }} onSubmit={handleSubmit}>
-                {({ errors, handleChange, isSubmitting, values }) => (
```

Comment:
Don't we need to disable the form submit button after submission and then while waiting for the `response`?

Reply:
`isSubmitting` has a known issue in `Formik` where it remains `true` when it's submit but doesn't change to `false` again. This is a known issue in `Formik`: https://github.com/formium/formik/issues/739. I can change this to an async function and use await, if we still want to use `isSubmitting`.